test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Query/Expressions/SelectExpandBinderTest.cs

```diff
@@ -344,9 +345,11 @@ public void ProjectAsWrapper_Element_ProjectedValueDoesNotContainInstance_IfSele
 
             // Assert
             Assert.Equal(ExpressionType.MemberInit, projection.NodeType);
-            Assert.NotEmpty((projection as MemberInitExpression).Bindings.Where(p => p.Member.Name == "Instance"));
+            Assert.Empty((projection as MemberInitExpression).Bindings.Where(p => p.Member.Name == "Instance"));
+            Assert.NotEmpty((projection as MemberInitExpression).Bindings.Where(p => p.Member.Name == "InstanceType"));
```

Comment:
Is it possible to verify the properties retrieved at ID and Orders? (From `$select=ID,Orders`)

Reply:
Or if the change has been verified in debugger, then that's fine.